apps/mosaico/src/Mosaico/Epaper.purs

```diff
@@ -36,7 +36,16 @@ component = do
     entitlements /\ setEntitlements <- useState' Nothing
     userAuth /\ setUserAuth <- useState' initialTokens
     useEffect (_.uuid <$> join user) do
-      when (isNothing $ join user) $ setEntitlements Nothing
+      when (isNothing $ join user) $ Aff.launchAff_ do
+        tokens <- User.loginIP paper
+        liftEffect case (hush tokens) of
+          Nothing -> setEntitlements $ Just mempty
+          Just {authToken, userId} -> do
+            let auth = {authToken, userId}
```

Comment:
This could just use `Just auth` in the pattern match since `authToken` and `userId` are otherwise unused.